pymeasure/instruments/fwbell/fwbell5080.py

```diff
@@ -63,6 +63,7 @@ class FWBell5080(Instrument):
         'amp-meter': 'DC:AM', 'amp-meter ac': 'AC:AM'
     }
     units = Instrument.control(
+        # TODO a newer version expects ':UNIT:FLUX:%s', is this right?
```

Comment:
possibly a typo - @cjermain could know.

Reply:
Let's leave the TODO in and wait for a reply from Colin, or a bug report by someone else.